apps/crn-server/test/entities/event.test.ts

```diff
@@ -205,6 +205,7 @@ describe('events entity', () => {
                 },
               ],
               role: 'Lead PI (Core Leadership)',
+              inactiveSinceDate: null,
```

Comment:
can you add a test where this date is actually returned? or maybe add this date to the fixtures rather than having a null